src/main/scala/com/johnsnowlabs/nlp/embeddings/BertSentenceEmbeddings.scala

```diff
@@ -391,13 +397,28 @@ class BertSentenceEmbeddings(override val uid: String)
 
   override def onWrite(path: String, spark: SparkSession): Unit = {
     super.onWrite(path, spark)
-    writeTensorflowModelV2(
-      path,
-      spark,
-      getModelIfNotSet.tensorflowWrapper,
-      "_bert_sentence",
-      BertSentenceEmbeddings.tfFile,
-      configProtoBytes = getConfigProtoBytes)
+
+    getEngine match {
+      case ModelEngine.tensorflow =>
+        writeTensorflowModelV2(
+          path,
+          spark,
+          getModelIfNotSet.tensorflowWrapper.get,
+          "_bert_sentence",
+          BertEmbeddings.tfFile,
```

Comment:
Shouldn't be here `BertSentenceEmbeddings`?

Reply:
Good catch, will fix it.